scripts/bumpPackages/cli.js

```diff
@@ -13,25 +9,13 @@ const promptVersion = async ({ packageNames }) => {
     type: 'select',
     name: 'version',
     choices: ['major', 'minor', 'patch'],
-    message: 'Choose the version you wish to bump',
+    message: 'Select the version',
   });
   return { packageNames, version };
 };
 
-initialPrompt('How would you like to enter which packages to bump?')
+initialPrompt('How would you like to choose which packages to version?')
```

Comment:
```suggestion
initialPrompt('Please choose which packages to version:')
```
Not sure I understand what prompt is saying it's current form?

Reply:
@andrewscfc it's because you have a choice of how to select a choice 😄 

<img width="431" alt="Screenshot 2021-03-18 at 08 41 28" src="https://user-images.githubusercontent.com/4798332/111597216-dcf0ae80-87c5-11eb-9f38-de29ae31ac4c.png">

Your suggestion sounds better and still makes sense anyway.